### PR TITLE
DOCS-6549 made changes

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

File: modules/ROOT/pages/api-gateway-encryption-mule4.adoc [hunk 1]
```diff
@@ -4,10 +4,9 @@ include::_attributes.adoc[]
 endif::[]
 
 
-Gateway startup encryption enables you to encrypt sensitive information stored by Mule runtime engine, such as policies, contracts, and the credentials required to configure the connection between your Mule application and Anypoint Platform.
-
-This feature is supported in Mule 4.2.0 and later.
+Gateway startup encryption enables you to encrypt sensitive information stored by Mule runtime engine, such as policies, contracts, and the credentials required to configure the connection between your Mule runtime engine (Mule) application and Anypoint Platform.
 
+This feature is supported in only Mule 4.2.0 and later. If a policy supporting encryption, such as Basic Authentication 1.2.0 is applied to an API that is running in Mule versions earlier than 4.2.0, the policy is not encrypted.
```
Comment (LorenaSoledad) [inline modules/ROOT/pages/api-gateway-encryption-mule4.adoc]:
```suggestion
This feature is supported only in Mule 4.2.0 and later. If a policy supporting encryption, such as Basic Authentication 1.2.0 is applied into an API that is running in Mule versions earlier than 4.2.0, the policy will not be encrypted.
```

About the example of Basic Auth, I don't know what is better: to give a particular example, or to give a reference to the `Policies That Support Encryption` section. I'll leave it up to you, but I had this new idea. 😃

**Review by LorenaSoledad — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/api-gateway-encryption-mule4.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/api-gateway-encryption-mule4.adoc]: We can give both right?

**Review by priyaayyar — COMMENTED** [review]
Added a link to the "Policies that support encryption" section.

File: modules/ROOT/pages/api-gateway-encryption-mule4.adoc [hunk 2]
```diff
@@ -4,10 +4,9 @@ include::_attributes.adoc[]
 endif::[]
 
 
-Gateway startup encryption enables you to encrypt sensitive information stored by Mule runtime engine, such as policies, contracts, and the credentials required to configure the connection between your Mule application and Anypoint Platform.
-
-This feature is supported in Mule 4.2.0 and later.
+Gateway startup encryption enables you to encrypt sensitive information stored by Mule runtime engine (Mule), such as policies, contracts, and the credentials required to configure the connection between your Mule application and Anypoint Platform.
 
+This feature is supported in Mule 4.2.0 and later. If a <<policies-that-support-encryption, policy that supports encryption>>, such as Basic Authentication 1.2.0, is applied to an API that is running in Mule versions earlier than 4.2.0, the policy is not encrypted.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/api-gateway-encryption-mule4.adoc]: Forgive my ignorance, but is a space allowed after the comma that defines the anchor in a <<anchor,text>> sequence? I always see it as not having a space, not sure if it works with one.

**Review by Mulesoft-Mysti — APPROVED** [review]
just a question about the space in the <<anchor,text>> command.

File: modules/ROOT/pages/api-gateway-encryption-mule4.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/api-gateway-encryption-mule4.adoc]: @Mulesoft-Mysti It does work and this is how I have used it. Let me know if I'm doing it all wrong.

Comment (LorenaSoledad) [inline modules/ROOT/pages/api-gateway-encryption-mule4.adoc]:
```suggestion
This feature is supported in Mule 4.2.0 and later. If a <<policies-that-support-encryption, policy that supports encryption>>, such as Basic Authentication 1.2.0, is applied to an API that is running in Mule versions earlier than 4.2.0, the policy configuration will not be encrypted.
```

**Review by LorenaSoledad — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/api-gateway-encryption-mule4.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/api-gateway-encryption-mule4.adoc]: Done! Thanks!

**Review by LorenaSoledad — APPROVED** [review]

**Pull request merged**